DigitalLearningSolutions.Data.Tests/DataServices/GroupsDataServiceTests.cs

```diff
@@ -211,6 +215,56 @@ public void GetRelatedProgressIdForGroupDelegate_returns_null_with_no_record()
             result.Should().BeNull();
         }
 
+        [Test]
+        public void UpdateGroupDescription_updates_record()

```

Comment:
Could we change this unit test name to
"TryUpdateGroupDescription_updates_record_with_correct_centreId" 
so that it matches the method name and circumstances.